site/src/pages/components/enhanced-range-input.explainer.mdx

```diff
@@ -0,0 +1,479 @@
+---
+menu: Active Proposals
+name: Enhanced Range Input (Explainer)
+layout: ../../layouts/ComponentLayout.astro
+---
+
+import RangeAnatomy from '../../components/range-anatomy'
+
+## Table of Contents
+
+- [Background](#background)
+- [Goals](#goals)
+- [Current State of Range Styling](#current-state-of-range-styling)
+- [Proposed Solution](#proposed-solution)
+    - [Standardized Anatomy](#standardized-anatomy)
+    - [Styling API](#styling-api)
+    - [Multi-Handle Support](#multi-handle-support)
+    - [Accessibility Enhancements](#accessibility-enhancements)
+    - [Progressive Enhancement](#progressive-enhancement)
+    - [Datalist Integration](#datalist-integration)
+    - [Multi-Color Range Segments](#multi-color-range-segments)
+- [Detailed Design](#detailed-design)
+    - [HTML Attributes](#html-attributes)
+    - [CSS Properties and Pseudo-elements](#css-properties-and-pseudo-elements)
+    - [JavaScript API](#javascript-api)
+    - [Examples](#examples)
+    - [Considerations and Open Questions](#considerations-and-open-questions)
+
+## Background
+
+The `<input type="range">` element has long been a part of HTML forms, providing a slider control for selecting numeric values. However, its current implementation lacks flexibility in styling and functionality, leading developers to often resort to custom implementations. These custom solutions can result in reduced performance, reliability, and accessibility compared to native form controls.
+
+The web development community has [expressed a need for more customizable and feature-rich range inputs](https://github.com/w3c/csswg-drafts/issues/4410#issuecomment-1087244943), particularly for use cases such as price range selectors, date/time range pickers, and other multi-value selection scenarios.
+
+- [Examples from CSSWG issue](https://github.com/w3c/csswg-drafts/issues/4410#issuecomment-1087244943)
+- [Examples from Open UI issue](https://github.com/openui/open-ui/issues/1165)
```

Comment:
I recommend linking out to the [parts analysis](https://open-ui.org/components/slider.research.parts/) I did a while back in Open UI